docs/Getting-Started/Create-bootable-media/create-bootable-usb.md

```diff
@@ -0,0 +1,36 @@
+# Create Pop!_OS Installation Media
+
+Pop!_OS can be run from a USB drive in a live environment. You can use the live USB to install Pop!_OS permenantly on a system, demo Pop!_OS without installing, or use the drive as a system recovery tool.
+
+---
+## Preparation
+
+Creating a bootable Pop!_OS USB drive requires three components: 
+
+- A USB flash drive. The flash drive must have at least 8 GB of free space. All data on the flash drive will be lost during the creation process.
+
+- The Pop!_OS.iso image: [Download Pop!_OS at this link.](https://pop.system76.com/)
+
+>**Note**: System76 offers their Pop!_OS.iso in 3 varieties: Intel/AMD, NVIDIA, and RAS PI. See [Choosing a Pop!_OS Image](create-bootable-usb.md#choose-a-pop_os-image).
```

Comment:
Wondering if talking about System76 should be "our" instead of "their". This is first-party documentation. If we don't want to talk in the first person, then reword to something like "The Pop!_OS iso image is available in 3 varieties."